src/renderer/sw_engine/tvgSwRenderer.cpp

```diff
@@ -87,6 +87,7 @@ struct SwShapeTask : SwTask
        Additionally, the stroke style should not be dashed. */
     bool antialiasing(float strokeWidth)
     {
+        if (rshape->stroke->antiAlias == ShapeRendering::CrispEdges) return false;
```

Comment:
> I'm not sure why unit tests are failing. For some reason, if I uncomment either line 90 or line 91 on tvgSwShape.cpp, the unit tests pass fine, but I don't understand why adding line 90 makes them fail.

When strokeWidth is 0, rshape->stroke is null. It looks like a null check for rshape->stroke is needed. 
```suggestion
        if (rshape->stroke && rshape->stroke->antiAlias == ShapeRendering::CrispEdges) return false;
```